app/assets/stylesheets/image_x.scss

```diff
@@ -147,6 +147,39 @@ $image-thumb-slider-background-color: $sul-background;
   text-align: center;
 }
 
+.#{$namespace}-image-x-restricted-thumb-container {
+  .#{$namespace}-image-x-auth-link {
+    @include rounded();
+    background: rgba($sul-background, 0.65);
+    bottom: 10px;
+    left: 10px;
+    padding: 5px;
+    position: absolute;
+  }
+
+  a {
+    @include stanford-only;
+  }
+}
+
+.#{$namespace}-image-x-iiif-drag-and-drop-link {
+  border-bottom: none !important;
+  cursor: move; // For non-webkit browsers
+  cursor: -webkit-grab;
```

Comment:
Using move above to get that behavior on browsers that don't support -webkit-grab.